doc/radosgw/layout.rst

```diff
@@ -26,19 +26,21 @@ metadata, bucket index, and data.
 We have 3 'sections' of metadata: 'user', 'bucket', and 'bucket.instance'.
 You can use the following commands to introspect metadata entries:
 
-$ radosgw-admin metadata list
-
-$ radosgw-admin metadata list bucket
-$ radosgw-admin metadata list bucket.instance
-$ radosgw-admin metadata list user
-
-$ radosgw-admin metadata get bucket:<bucket>
-$ radosgw-admin metadata get bucket.instance:<bucket>:<bucket_id>
-$ radosgw-admin metadata get user:<user>   # get or set
-
-user: Holds user information
-bucket: Holds a mapping between bucket name and bucket instance id
-bucket.instance: Holds bucket instance information[2]
+.. code:: bash
+
+    $ radosgw-admin metadata list
+    $ radosgw-admin metadata list bucket
+    $ radosgw-admin metadata list bucket.instance
+    $ radosgw-admin metadata list user
+
+    $ radosgw-admin metadata get bucket:<bucket>
+    $ radosgw-admin metadata get bucket.instance:<bucket>:<bucket_id>
+    $ radosgw-admin metadata get user:<user>   # get or set
+    
+    # comments:
```

Comment:
I have tried this, but it couldn't be sub-list and mixs with `metadata/bucket index/data`. how about  enumerated list. like

1. user: Holds user information
2. bucket: Holds a mapping between bucket name